From mack@erwied.de  Fri Jun  2 13:49:57 2000
X-VM-v5-Data: ([nil nil nil nil nil nil nil nil nil]
	["816" "Friday" "2" "June" "2000" "21:21:38" "+0200" "Jochen Erwied" "jochen@erwied.de" nil "21" "[Bug?] Flexbackup 0.9.8 and levels > 0" "^From:" nil nil "6" nil nil nil nil nil nil nil nil]
	nil)
Return-Path: <mack@erwied.de>
Received: from localhost (IDENT:root@localhost [127.0.0.1])
	by cx81598-a.chnd1.az.home.com (8.9.3/8.9.3) with ESMTP id NAA25309
	for <edwinh+flexbackup@localhost>; Fri, 2 Jun 2000 13:49:57 -0700
Received: from mail.chnd1.az.home.com
	by localhost with POP3 (fetchmail-5.3.1)
	for edwinh+flexbackup@localhost (single-drop); Fri, 02 Jun 2000 13:49:57 -0700 (MST)
Received: from h7.mail.home.com ([24.0.95.42]) by mail.rdc1.az.home.com
          (InterMail vM.4.01.02.00 201-229-116) with ESMTP
          id <20000602204445.JWHZ11091.mail.rdc1.az.home.com@h7.mail.home.com>
          for <flexbackup@mail.chnd1.az.home.com>;
          Fri, 2 Jun 2000 13:44:45 -0700
Received: from mx6-rwc.mail.home.com (mx6-rwc.mail.home.com [24.0.95.35])
	by h7.mail.home.com (8.9.3/8.9.0) with ESMTP id NAA01690
	for <flexbackup@home.com>; Fri, 2 Jun 2000 13:45:04 -0700 (PDT)
Received: from mail.erwied.de (joker.e.ruhr.de [141.39.230.20])
	by mx6-rwc.mail.home.com (8.9.1/8.9.1) with ESMTP id NAA20996
	for <flexbackup@home.com>; Fri, 2 Jun 2000 13:44:43 -0700 (PDT)
Received: from mack by mail.erwied.de with local (Exim 3.14 #3)
	id 12xx0k-00065i-00
	for flexbackup@home.com; Fri, 02 Jun 2000 21:21:38 +0200
Message-ID: <20000602212136.A23378@joker.erwied.de>
Mime-Version: 1.0
Content-Type: text/plain; charset=us-ascii
Content-Disposition: inline
Status: O
Content-Length: 816
Lines: 21
From: Jochen Erwied <jochen@erwied.de>
To: flexbackup@home.com
Subject: [Bug?] Flexbackup 0.9.8 and levels > 0
Date: Fri, 2 Jun 2000 21:21:38 +0200
Reply-To: Jochen Erwied <mack@joker.e.ruhr.de>
User-Agent: Mutt/1.3.2i

When doing a 'flexbackup -fs all -level 9' a second time, flexbackup takes
the dates of the last level-9 backup instead of any lower-numbered backup
(level-0 in this case).

Fix appended.

-- 
Jochen Erwied      @home: jochen@erwied.de     +49-208-38800-18, FAX: -19
Sauerbruchstr. 17  @work: joe@mbs-software.de  +49-2151-7294-24, FAX: -50
D-45470 Muelheim   All addresses in this article are valid and replyable!

---snip
--- ../linux/flexbackup-0.9.8/flexbackup        Wed Nov  3 04:13:41 1999
+++ flexbackup  Fri Jun  2 21:12:50 2000
@@ -1141,5 +1141,5 @@
     foreach $file (reverse sort @files) {
         next if ($file !~ m/^$cfg::sprefix$label\.([0-9])$/);
-        if ($1 <= $main::level) {
+        if ($1 < $main::level) {
             $targetfile = "$cfg::stampdir/$file";
             $lastlevel = $1;

